documentation/architecture/cloud/gsuite/accounts.md

```diff
@@ -0,0 +1,24 @@
+# Helpful Engineering G Suite - Account Policy
+
+## Types
+Accounts can be divided in two major categories: personal accounts, external accounts and purpose-specific accounts. The latter should be avoided whenever possible lest losing fine-grained control over individual permissions, but they're required for some specific use cases like Google Analytics and common file storage.
```

Comment:
```suggestion
Accounts can be divided in three major categories: personal accounts, external accounts and shared (purpose-specific) accounts. Shared accounts will be created only in very specific cases where several individuals need access to a single account for access to G Suite managed resources such as Google Analytics and common file storage.
```

Reply:
Nobody expects the Spanish inquisition. :smile: Thanks!